Engine/Source/ModuleDebugDraw.h

```diff
@@ -21,12 +21,11 @@ class ModuleDebugDraw : public Module
 	bool            CleanUp();
 
     void Draw(const float4x4& viewproj, unsigned width, unsigned height);
-    void DrawBoundingBox(const OBB& obb);
-    void DrawQuadtree(const AABB& aabb);
     void DrawAxis();
     void DrawFrustum(const Frustum& frustum);
     bool GetShouldRenderGrid() const { return mDrawGrid; }
-    void SetRenderGrid(bool a) { mDrawGrid = a; }     
+    void SetRenderGrid(bool a) { mDrawGrid = a; }
+    void DrawCube(const OBB& obb, const float3& color);
     void DrawSphere(const float center[3], const float color[3], const float radius);
     void DrawCone(const float pos[3], const float dir[3], const float color[3], const float bRadius);
     void DrawLine(const float3& position, const float3& direction, const float3& color);
```

Comment:
I think that is a good idea instead of having a DrawCube method and a DrawBoundingBox method, to have a two DrawBoundingBox one for OBB and one for AABB and adding the color as a parameter as you have in DrawCube.